### PR TITLE
Music Working, Plays while game is running, and Loops

### PR DESCRIPTION
**Pull request by NathanGr33n**
Added Music Class and Audio subfolder inside Content. Music is playing, waiting on dope soundtrack.

### CODE REVIEW

File: TKGame/Art.cs [hunk 1]
```diff
@@ -21,11 +21,11 @@ static class Art
...
-            BackgroundTexture = content.Load<Texture2D>(@"Art/Cobble"); //Background image
-            EnemyTexture = content.Load<Texture2D>(@"Art/DoomguyRightFacing"); // enemy as doomguy
-            ItemTexture = content.Load<Texture2D>(@"Art/DiamondSwordLeftTilt"); // item as diamond sword 
+            PlayerTexture = content.Load<Texture2D>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Art/Player"); // Will need to be modified when we create a better structure for our files.
+            WeaponTexture = content.Load<Texture2D>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Art/Weapons/IronSword"); //Sword Weapon Image    
+            BackgroundTexture = content.Load<Texture2D>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Art/Cobble"); //Background image
+            EnemyTexture = content.Load<Texture2D>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Art/DoomguyRightFacing"); // enemy as doomguy
+            ItemTexture = content.Load<Texture2D>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Art/DiamondSwordLeftTilt"); // item as diamond sword
```
Comment (keeganhauser) [inline TKGame/Art.cs:28]: smh with the absolute paths

File: TKGame/Music.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+        /// <summary>
+        /// Loads up audio and plays it. Takes a Content Manager and a float volume value.
+        /// </summary>
+        /// <param name="Content"></param>
+        /// <param name="vol"></param>
+        public static void LoadContent(ContentManager Content, float vol)
+        {
+            //Song is just temporary until Keegan makes us a basedAF soundtrack
+            backgroundSong = Content.Load<Song>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Audio/Break-Out-Loop");
```
Comment (keeganhauser) [inline TKGame/Music.cs]: 😡

**Review by keeganhauser — CHANGES_REQUESTED** [review]
fix your hard coded paths nerd

**Review by keeganhauser — APPROVED** [review]

File: TKGame/Music.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        #region Member Variables
+        private static Song backgroundSong;
+        #endregion
+
+        /// <summary>
+        /// Loads up audio and plays it. Takes a Content Manager and a float volume value.
+        /// </summary>
+        /// <param name="Content"></param>
+        /// <param name="vol"></param>
+        public static void LoadContent(ContentManager Content, float vol)
```
Comment (NateCastleOIT) [inline TKGame/Music.cs:23]: We probably want to be able to change the volume with a function call or something as well

**Review by NateCastleOIT — APPROVED** [review]
LG

File: TKGame/Music.cs (hunk 3 above)
Comment (NathanGr33n) [reply TKGame/Music.cs:23]: You guys don't have volume controls on your keyboard???

**Pull request merged**